### PR TITLE
Output service logs in `TRACE` while service is starting. Closes #810

### LINKED ISSUE #810: Trace log output of service if it fails to start.

### PR DESCRIPTION
**Pull request by binaek89**

### CODE REVIEW

File: db/local_db/start_database.go [hunk 1]
```diff
@@ -143,6 +145,11 @@ func startPostgresProcess(port int, listen StartListenType, invoker constants.In
 	utils.LogTime("startPostgresProcess start")
 	defer utils.LogTime("startPostgresProcess end")
 
+	logChannel := make(chan string, 1000)
+	defer func() {
```
Comment (kaidaguerre) [inline db/local_db/start_database.go:149]: defer close(logChannel)

File: db/local_db/start_database.go [hunk 2]
```diff
@@ -221,11 +228,33 @@ func startPostgresProcess(port int, listen StartListenType, invoker constants.In
...
+		return err
+	}
+	defer stopCollectionFn()
+
+	err = postgresCmd.Start()
 	if err != nil {
 		return err
 	}
 
+	go func() {
```
Comment (kaidaguerre) [inline db/local_db/start_database.go:242]: this is already a massive function - factor this into a separate function
```
go displayServiceLogs()
```

**Review by kaidaguerre — CHANGES_REQUESTED** [review]

File: db/local_db/start_database.go (hunk 1 above)
Comment (binaek89) [reply db/local_db/start_database.go]: fixed

File: db/local_db/start_database.go (hunk 2 above)
Comment (binaek89) [reply db/local_db/start_database.go]: fixed

**Pull request merged**